samza-sql/src/main/java/org/apache/samza/sql/translator/QueryTranslator.java

```diff
@@ -79,7 +79,7 @@
   private final SamzaSqlApplicationConfig sqlConfig;
   private final StreamApplicationDescriptor streamAppDescriptor;
   private final Map<String, DelegatingSystemDescriptor> systemDescriptors;
-  private final Map<String, MessageStream<KV<Object, Object>>> inputMsgStreams;
+  private final Map<String, MessageStream<Object>> inputMsgStreams;
```

Comment:
Should this be of type Map<String, MessageStream<SamzaSqlMessage>>?

Reply:
it could. I changed it to Map<String, MessageStream<SamzaSqlInputMessage>>